build.gradle.kts

```diff
@@ -11,5 +13,34 @@ subprojects {
     group = GROUP
     version = VERSION_NAME
 
-    // TODO @jatwood re-add ktlint via a plugin (e.g. spotless)
+    apply(plugin = "com.diffplug.spotless")
+    extensions.configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+        kotlin {
+            target("**/*.kt")
+            targetExclude("$buildDir/**/*.kt")
+            ktlint()
+                .editorConfigOverride(
+                    mapOf(
+                        "ktlint_standard_multiline-expression-wrapping" to "disabled",
+                    ),
+                )
+            trimTrailingWhitespace()
+            indentWithSpaces()
+            endWithNewline()
+            licenseHeader("// Copyright Block, Inc.")
```

Comment:
I think this is actually in conflict with the actual license in the project. It currently uses APACHE ( https://github.com/afterpay/sdk-android/blob/master/LICENSE).
I recall going trough this with J Wilson, here's an example of what it should probably look like: https://github.com/cashapp/cash-app-pay-android-sdk/blob/main/build.gradle#L47

might need to tweak the entity, but you get the gist